packages/main/src/Calendar.ts

```diff
@@ -246,10 +159,94 @@ const metadata = {
  * @public
  * @since 1.0.0-rc.11
  */
+@customElement("ui5-calendar")
+@fastNavigation
+/**
+ * Fired when the selected dates change.
+ * <b>Note:</b> If you call <code>preventDefault()</code> for this event, the component will not
+ * create instances of <code>ui5-date</code> for the newly selected dates. In that case you should do this manually.
+ *
+ * @event sap.ui.webc.main.Calendar#selected-dates-change
+ * @allowPreventDefault
+ * @param {Array} values The selected dates
+ * @param {Array} dates The selected dates as UTC timestamps
+ * @public
+ */
+@event("selected-dates-change", {
+	detail: {
+		dates: { type: Array },
+		values: { type: Array },
+	},
+})
+
+@event("show-month-press")
+@event("show-year-press")
 class Calendar extends CalendarPart {
-	static get metadata() {
-		return metadata;
-	}
+	/**
+	 * Defines the type of selection used in the calendar component.
+	 * Accepted property values are:<br>
+	 * <ul>
+	 * <li><code>CalendarSelectionMode.Single</code> - enables a single date selection.(default value)</li>
+	 * <li><code>CalendarSelectionMode.Range</code> - enables selection of a date range.</li>
+	 * <li><code>CalendarSelectionMode.Multiple</code> - enables selection of multiple dates.</li>
+	 * </ul>
+	 * @type {sap.ui.webc.main.types.CalendarSelectionMode}
+	 * @defaultvalue "Single"
+	 * @public
+	 */
+	@property({
+		type: CalendarSelectionMode,
+		defaultValue: CalendarSelectionMode.Single,
+	})
+	selectionMode!: CalendarSelectionMode;
+
+	/**
+	 * Defines the visibility of the week numbers column.
+	 * <br><br>
+	 *
+	 * <b>Note:</b> For calendars other than Gregorian,
+	 * the week numbers are not displayed regardless of what is set.
+	 *
+	 * @type {boolean}
+	 * @name sap.ui.webc.main.Calendar.prototype.hideWeekNumbers
+	 * @defaultvalue false
+	 * @public
+	 */
+	@property({ type: Boolean })
+	hideWeekNumbers!: boolean;
+
+	/**
+	 * Which picker is currently visible to the user: day/month/year
+	 */
+	 @property({ defaultValue: "day" })
+	_currentPicker!: string;
+
+	@property({ type: Boolean })
+	_previousButtonDisabled!: boolean;
+
+	@property({ type: Boolean })
+	_nextButtonDisabled!: boolean;
+
+	@property()
+	_headerMonthButtonText?: string;
+
+	@property()
+	_headerYearButtonText?: string;
+
+	@property()
+	_headerYearButtonTextSecType?: string;
+
+	/**
+	 * Defines the selected date or dates (depending on the <code>selectionMode</code> property)
+	 * for this calendar as instances of <code>ui5-date</code>.
+	 *
+	 * @type {sap.ui.webc.main.ICalendarDate[]}
+	 * @name sap.ui.webc.main.Calendar.prototype.dates
```

Comment:
previously this slot was named as default in API reference. I think @name should point sap.ui.webc.main.Calendar.prototype.default